assets/src/blocks/HubspotForm/useDynamicHeight.js

```diff
@@ -0,0 +1,34 @@
+import { useState, useEffect } from 'react';
+
+export const useDynamicHeight = (wrapperRef) => {
```

Comment:
Can you explain the motivation for this hook? It's not obvious why this is needed, or if it's needed at all.

The current usage sets the height on an element, where the passed in `wrapperRef` is the only descendant of that element. Doesn't that mean the parent element already has the right height?

Reply:
You're are right, I've removed it. 
This hook has been used before, now it's not necessary for the current implementation.